deploy-all.sh

```diff
@@ -0,0 +1,41 @@
+mkdir -p ./temp
+
+if [[ -z "${NAMESPACE_TO_GENERATE}" ]]; then
+    export NAMESPACE_TO_GENERATE="tnf"
+fi
+
+export res=$(oc get namespace $NAMESPACE_TO_GENERATE 2>&1)
+if [[ ${res#Error from server (NotFound):} != ${res} ]]; then
+    cat ./local-test-infra/namespace.yaml | ./script/mo > ./temp/rendered-namespace.yaml
+    oc apply -f ./temp/rendered-namespace.yaml
+    rm ./temp/rendered-namespace.yaml
+else
+    echo "namespace ${NAMESPACE_TO_GENERATE} already exists, no reason to recreate"
+fi
+
+export res=$(oc get pod -A -l test-network-function.com/generic=fs_diff_master 2>&1)
```

Comment:
Use NAMESPACE_TO_GENERATE instead of "-A" to find the existing pods. Currently, if an test or partner pod is created in any namespace, it will prevent creation of the test /partners pods in the new namespace.